apps/settings/js/media_storage.js

```diff
@@ -538,7 +538,7 @@ require([
       this.defaultMediaLocation.addEventListener('click', this);
       this.makeDefaultLocationMenu();
 
-      window.addEventListener('localized', this);
+      document.addEventListener('DOMRetranslated', this);
```

Comment:
I'm afraid we didn't get to document the L20n API yet :( Basically, the only event we fire now is `DOMRetranslated` on document and we do this only for consecutive language changes.

For the first one you have `document.l10n.ready` promise if you need it, but since all operations are async, you can just call you `setAttributes` or `formatValue` without waiting for anything.